### PR TITLE
Update to Electron 14

### PR DESCRIPTION
**Pull request by dfunckt**
Depends on https://github.com/balena-io/etcher/pull/4000

### CODE REVIEW

**Comment by builder555** [comment]
Tested this version locally and on EtcherPro, it seems to have a regression on #3792

**Comment by dfunckt** [comment]
It’s unlikely this PR is at fault @builder555, can you test [v1.15.1](https://github.com/balena-io/etcher/releases/tag/v1.15.1) instead? That’s far more likely to have broken something.

**Comment by builder555** [comment]
just tested it - 1.15.1 works fine with a CM4 @dfunckt

**Comment by dfunckt** [comment]
That’s actually great news. There’s nothing that immediately comes to mind about what might have caused the regression but the update wasn’t straightforward either, so I may have missed something. I’ll take another tomorrow and reach out in  Zulip.

In the meantime, @builder555 can you please give #4000 a go too? That would narrow down things further.

**Comment by mcraa** [comment]
Fin + CM3 or CM4s does not mount on windows. :/

File: lib/gui/etcher.ts [hunk 1]
```diff
@@ -209,6 +210,7 @@ async function createMainWindow() {
 	);
 
 	const page = mainWindow.webContents;
+	remoteMain.enable(page);
```
Comment (mcraa) [inline lib/gui/etcher.ts:214]: on line 233, `allowRendererProcessReuse`, that property is removed: https://www.electronjs.org/blog/electron-14-0/#removed-appallowrendererprocessreuse

does this remoteMain add that back, or we can delete that line? (same for tests)

Comment (dfunckt) [reply lib/gui/etcher.ts:214]: Yeah, we should remove the flag, I expect it has no effect but for some reason typings allow it to be set which is misleading.

**Comment by dfunckt** [comment]
I self-certify!

**Pull request merged**